src/tbcd.md

```diff
@@ -14,6 +14,8 @@ head:
 - Token Bonding Curve Dollars (TBCD) is a newly proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve
 - TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low
 - TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via an on-chain referendum decided by all XOR token holders
+- 10% of the reminted [PSWAP](./pswap) is allocated to buyback-and-burn TBCD. Similarly, 10% of the reminted [VAL](./val) is allocated to buyback-and-burn TBCD.
```

Comment:
```suggestion
- 10% of the reminted [PSWAP](./pswap) is allocated to buy back-and-burn TBCD. Similarly, 10% of the reminted [VAL](./val) is allocated to buyback-and-burn TBCD.
```

Reply:
I think the point of this change was to use the term *buyback* consistently throughout the doc, no?